homeassistant/components/luxtronik/__init__.py

```diff
@@ -0,0 +1,140 @@
+"""Support for Luxtronik heatpump controllers."""
+from datetime import timedelta
+import logging
+
+from luxtronik import LOGGER as LuxLogger, Luxtronik as Lux
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_PORT
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+LuxLogger.setLevel(level="WARNING")
+
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_PARAMETER = "parameter"
+ATTR_VALUE = "value"
+
+CONF_INVERT_STATE = "invert"
+CONF_SAFE = "safe"
+CONF_GROUP = "group"
+CONF_PARAMETERS = "parameters"
+CONF_CALCULATIONS = "calculations"
+CONF_VISIBILITIES = "visibilities"
+
+CONF_CELSIUS = "celsius"
+CONF_SECONDS = "seconds"
+CONF_PULSES = "pulses"
+CONF_IPADDRESS = "ipaddress"
+CONF_TIMESTAMP = "timestamp"
+CONF_ERRORCODE = "errorcode"
+CONF_KELVIN = "kelvin"
+CONF_BAR = "bar"
+CONF_PERCENT = "percent"
+CONF_RPM = "rpm"
+CONF_ENERGY = "energy"
+CONF_VOLTAGE = "voltage"
+CONF_HOURS = "hours"
+CONF_FLOW = "flow"
+CONF_LEVEL = "level"
+CONF_COUNT = "count"
+CONF_VERSION = "version"
+
+TIME_SECONDS = "s"
+TIME_HOUR = "h"
+TEMP_KELVIN = "K"
+PERCENTAGE_PERCENT = "%"
+VOLTAGE_VOLT = "V"
+FLOW_LITERS_PER_MINUTE = "l/min"
```

Comment:
Since you have so many constants you could think about moving them into a separate const.py file.
Moreover, I would try to reuse as many constants from const.py instead of introducing new constants.

Reply:
Sure, I'll move them into a sepearte const.py file. I already tried to reuse constants from const.py but there are many that do not exists. 

Do you think I should update the global const file with not yet existing values (like kelvin for temperatures and so on)?

Reply:
I would not recommend to update the global const file since it should only be updated if more integrations use these constants.
Therefore, please keep these constants local to your integration and try to minimize them as much as possible.